homeassistant/components/hisense_aehw4a1/climate.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, device):
         self._preset_modes = PRESET_MODES
         self._available = None
         self._on = None
-        self._temperature_unit = None
+        self._attr_temperature_unit = None
```

Comment:
`temperature_unit` should never be `None`, so I think it was a bug.
Should we instead use a default?

Reply:
It updates before add, so it can indeed never be None.

I think you can just remove this line.

Reply:
```suggestion
```